envoy/upstream/upstream.h

```diff
@@ -836,11 +836,10 @@ class ClusterInfo : public Http::FilterChainFactory {
   }
 
   /**
-   * @return const envoy::config::cluster::v3::LoadBalancingPolicy_Policy& the load balancing policy
-   * to use for this cluster.
+   * @return const ProtobufWkt::Message& the validated load balancing policy configuration to use
+   * for this cluster.
    */
-  virtual const envoy::config::cluster::v3::LoadBalancingPolicy_Policy&
-  loadBalancingPolicy() const PURE;
+  virtual const ProtobufTypes::MessagePtr& loadBalancingPolicy() const PURE;
 
```

Comment:
Why we need this change?
Load balancer will be created at runtime in every worker. We shouldn't use non-validated raw `Policy` to create load balancers. Uncaught exceptions may be throwed.  
So, we should unpack and validate this typed config in the configure phase.